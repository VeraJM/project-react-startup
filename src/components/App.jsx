import React from 'react';
import styled from 'styled-components';
import Banner from './Banner';
import Wrapper from './Wrapper';
import AccountSection from './AccountSection';
import TransactionsSection from './TransactionsSection';

const Table = styled.table`
  border-collapse: collapse;
  border: 1px solid black;
  width: 100%;
  height: 80%;
`;

export default function App() {
  return (
    <Wrapper>
      <Banner />
      <Table>
        <tbody>
          <tr>
            <td width="30%"><AccountSection /></td>
            <td><TransactionsSection /></td>
          </tr>
        </tbody>
      </Table>
    </Wrapper>
  );
}
