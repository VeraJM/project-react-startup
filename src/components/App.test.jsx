import React from 'react';
import toJson from 'enzyme-to-json';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import App from './App';

Enzyme.configure({ adapter: new Adapter() });

describe('App', () => {
  it('renders', () => {
    const app = shallow(<App />);

    expect(toJson(app)).toMatchSnapshot();
  });
});
