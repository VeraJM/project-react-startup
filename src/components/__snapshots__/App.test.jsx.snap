// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div
  className="game"
>
  <div
    className="game-board"
  >
    <styled.h1
      primary={true}
    >
      Operando Primary
    </styled.h1>
    <styled.h1>
      Operando Secondary
    </styled.h1>
    <Board
      onClick={[Function]}
      squares={
        Array [
          null,
          null,
          null,
          null,
          null,
          null,
          null,
          null,
          null,
        ]
      }
    />
  </div>
  <div
    className="game-info"
  >
    <div>
      Next player: X
    </div>
    <ol>
      <li
        key="0"
      >
        <button
          onClick={[Function]}
        >
          Go to game start
        </button>
      </li>
    </ol>
  </div>
</div>
`;
